pkg/velaql/providers/query/endpoint.go

```diff
@@ -108,7 +109,7 @@ func getServiceEndpoints(ctx context.Context, cli client.Client, gvk schema.Grou
 	switch gvk.Kind {
 	case "Ingress":
 		if gvk.Group == networkv1beta1.GroupName && (gvk.Version == "v1beta1" || gvk.Version == "v1") {
-			var ingress networkv1beta1.Ingress
+			var ingress v1.Ingress
```

Comment:
Do we need to support versions before v1.19?

Reply:
Discussed with @wonderflow, we think it's needn't.